ci/generate-docs.sh

```diff
@@ -4,8 +4,9 @@ set -e -u -x -o pipefail
 
 cd "$(dirname "$0")/../"
 
+DOCGEN_PIN="${DOCGEN_OVERRIDE:-v1.0}"
 CURRENT_DIR=$(pwd)
-TMP_DIR=$(mktemp -d)
+TMP_DIR=$(mktemp -d "${TMPDIR:-/tmp}/XXXXXXXXX")
```

Comment:
xxxx?

Reply:
See https://github.com/spatialos/gdk-for-unity/pull/1328#discussion_r396390594